Taskfile.yml

```diff
@@ -129,6 +131,50 @@ tasks:
       - "{{ .MULTIMOD_BIN }} prerelease --all-module-sets --skip-go-mod-tidy=true --commit-to-different-branch=false"
       - git commit --amend --signoff --no-edit
 
+  ##
+  ## Helm
+  ##
+  helm:dependency-update:
+    desc: Update Helm chart dependencies
+    deps:
+      - task: deps:helm
+    cmds:
+      - cmd: echo "Updating Helm chart dependencies..."
+      - cmd: "{{ .HELM_BIN }} dependency update helm/oasf-sdk"
+
+  helm:lint:
+    desc: Lint Helm chart
+    deps:
+      - task: deps:helm
+    cmds:
+      - cmd: echo "Linting Helm chart..."
+      - cmd: "{{ .HELM_BIN }} lint helm/oasf-sdk --with-subcharts"
+
+  helm:package:
+    desc: Package Helm chart
+    deps:
+      - task: deps:helm
+    vars:
+      VERSION: '{{ .VERSION | default "0.0.1-dev" }}'
+    cmds:
+      - cmd: echo "Packaging Helm chart version {{.VERSION}}"
+      - cmd: "{{ .HELM_BIN }} package helm/oasf-sdk --dependency-update --version {{.VERSION}}"
+      - cmd: echo "Chart packaged successfully as oasf-sdk-{{.VERSION}}.tgz"
+
+  helm:push:
+    desc: Push Helm chart to OCI registry
+    deps:
+      - task: deps:helm
+    vars:
+      VERSION: '{{ .VERSION | default "0.0.1-dev" }}'
+    preconditions:
+      - sh: test -f oasf-sdk-{{.VERSION}}.tgz
+        msg: "Chart package oasf-sdk-{{.VERSION}}.tgz not found. Run 'task helm:package VERSION={{.VERSION}}' first."
+    cmds:
+      - cmd: echo "🚧 Pushing oasf-sdk-{{.VERSION}}.tgz to oci://ghcr.io/agntcy/oasf-sdk/helm-charts"
+      - cmd: "{{ .HELM_BIN }} push oasf-sdk-{{.VERSION}}.tgz oci://ghcr.io/agntcy/oasf-sdk/helm-charts"
+      - cmd: echo "✅ Chart pushed successfully to oci://ghcr.io/agntcy/oasf-sdk/helm-charts"
```

Comment:
I'd put `oci://ghcr.io/agntcy/oasf-sdk/helm-charts` at the top of the file in a variable